### DISCUSSION TITLE
課題11についての質問

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#36] [Q&A]
https://github.com/nsc-kiwamu/traning-takeda/pull/34#issue-1309190375

↑上記のリンク先にて、質問を書かせていただいております。
よろしくお願いいたします。

**Comment by nsc-kiwamu** [comment]
https://github.com/nsc-kiwamu/traning-takeda/pull/34#issuecomment-1189112658
